diff --git a/emscripten/2_build_toolchain.sh b/emscripten/2_build_toolchain.sh
index 8908b2a..a9bc43d 100755
--- a/emscripten/2_build_toolchain.sh
+++ b/emscripten/2_build_toolchain.sh
@@ -9,12 +9,7 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $SCRIPT_DIR/../shared/import.sh
 
 # Number of CPU
-os=`uname`
-if [ $os = "Darwin" ] ; then
-	nproc=$(getconf _NPROCESSORS_ONLN)
-else
-	nproc=$(nproc)
-fi
+readonly nproc=1
 
 # no ccache support currently with em* wrappers
